website/content/docs/concepts/integrated-storage/autopilot.mdx

```diff
@@ -55,17 +55,33 @@ API](/api-docs/system/storage/raftautopilot#set-configuration) to manage its
 behavior. Autopilot gets initialized with the following default values.
 
 - `cleanup_dead_servers` - `false`
+  - This controls whether to remove dead servers from
+    the Raft peer list periodically or when a new server joins. This requires that
+    `min-quorum` is also set.
 
 - `dead_server_last_contact_threshold` - `24h`
+  - Limit on the amount of time
+    a server can go without leader contact before being considered failed. This
+    takes effect only when `cleanup_dead_servers` is set.
 
 - `min_quorum` - This doesn't default to anything and will need to be set to at
   least 3 when `cleanup_dead_servers` is set as `true`.
+  - Minimum number of servers allowed in a cluster before
```

Comment:
I don't think that's quite correct, isn't it rather that autopilot will never reduce the number of voters below this threshold?

Reply:
Yes, but I think what you're saying and what this is saying are the same thing. Anyway, the purpose here was to bring these two separate pages in sync with each other. I just copy/pasted from one page to the other without editing anything.

Reply:
Sure, but I was having a slack convo with someone just yesterday who didn't understand what this does, so I thought this was a good occasion to improve our docs for this feature.

The phrasing "minimum number of servers allowed in a cluster before autopilot can prune dead servers" makes it sound like if you set it to 3, so long as you have 3 nodes in your cluster, autopilot may decide to remove one it considers dead.

Also saying "should be at least 3" isn't very helpful, we should probably say something more like "this should be the expected number of voters in your cluster".

Reply:
Fixed in https://github.com/hashicorp/vault/pull/17397/commits/f43fc264e0168c3fefd8cf15009a7bc25c5be596